CHANGELOG.md

```diff
@@ -9,8 +9,29 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Added
 
 ### Fixed
-- fix: queries with up to 65535 (inclusive) parameters are supported now (previous limit was 32767) [PR #2525](https://github.com/pgjdbc/pgjdbc/pull/2525)
-- fix: workaround JarIndex parsing issue by using groupId/artifactId-version directory namings. Regression since 42.2.13. [PR #2531](https://github.com/pgjdbc/pgjdbc/pull/2531), [issue #2527](https://github.com/pgjdbc/pgjdbc/issues/2527)
+
+[42.4.0] (2022-06-09 08:14:02 -0400)
```

Comment:
These lines look odd. Are they intentional?

Reply:
Ya, when I change the date of the release, I have to change the time as well

Reply:
ah, right, thanks, my groovy code is still a WIP